### PR TITLE
Release

### PR DESCRIPTION
**Pull request by paulinebessoles**
* Fix rubocop

* Don't query deleted users

* Don't query empty email users

### CODE REVIEW

**Review by eliegaboriau — APPROVED** [review]

**Comment by paulinebessoles** [comment]
Waiting for refactor before merging

File: lib/decidim/cleaner/version.rb [hunk 1]
```diff
@@ -5,11 +5,11 @@ module Decidim
   # This holds the decidim-meetings version.
   module Cleaner
     def self.version
-      "1.0.1"
+      "4.0.0"
```
Comment (armandfardeau) [inline lib/decidim/cleaner/version.rb:8]:
```suggestion
      "4.1.0"
```

4.0.0 has already been released

**Review by armandfardeau — CHANGES_REQUESTED** [review]

**Comment by armandfardeau** [comment]
Hi @paulinebessoles It seems that a rebase was needed and therefore this P.R. must be closed, feel free to re-open when #21 is merged.

**Comment by paulinebessoles** [comment]
Ok, thank you for the update!

**Pull request closed without merging**